Index: src/GameQuestions
===================================================================
diff --git a/src/GameQuestions b/src/GameQuestions
new file mode 100644
--- /dev/null	
+++ b/src/GameQuestions	
@@ -0,0 +1,46 @@
+
+Table for GameQuestions
+Insert into (number, question)
+
+1. Have i ever smoked a cigarette;
+
+2. Have i ever cheated;
+
+3. Have i ever been drunk on a weekday;
+
+4. Have i ever downloaded a dating app;
+
+5. Have i ever eaten a whole pizza by myself;
+
+6. Have i ever done anything illegal;
+
+7. Have i ever parked in a disabled space;
+
+8. Have i ever thrown up on another person;
+
+9. Have i ever been handcuffed;
+
+10. Have i ever eaten food that fell on the floor;
+
+11. Have i ever had a one night stand;
+
+12. Have i ever cut my own hair;
+
+13. Have i ever sent a naughty message;
+
+14. Have i ever put on music at home and danced alone;
+
+15. Have i ever dyed my hair;
+
+16. Have i ever had a tattoo;
+
+17. Have i ever looked at another person's messages;
+
+18. Have i ever seen Game of Thrones;
+
+
+
+
+
+
+
