Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.io.IOException;\n\npublic class Main {\n\n    public static void main(String[] args) throws IOException {\n        // Here we call the method to welcome the player and log in or create an account.\n       TextUI textUI = new TextUI();\n        textUI.startGame();\n        textUI.gameSettup();\n\n        //Here we call the method to start the game\n       // QuestionsDB questionsDB  = new QuestionsDB();\n        //questionsDB.getAllQuestions();\n\n        // Here we call the method to save the players points and display them\n        //Score score = new Score();\n        //score.savePoint();\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision 639ae1e58a7023684c48a5cf1a0347c69786d631)
+++ b/src/Main.java	(date 1670932402265)
@@ -9,8 +9,8 @@
         textUI.gameSettup();
 
         //Here we call the method to start the game
-       // QuestionsDB questionsDB  = new QuestionsDB();
-        //questionsDB.getAllQuestions();
+       QuestionsDB questionsDB  = new QuestionsDB();
+        questionsDB.getAllQuestions();
 
         // Here we call the method to save the players points and display them
         //Score score = new Score();
